test/__tests__/uppy-server.js

```diff
@@ -0,0 +1,65 @@
+/*global jest:false, test:false, expect:false, describe:false*/
```

Comment:
You can add `jest: true` to the eslintrc so that these exceptions aren't necessary anymore and can be removed.

---

I personally recently turned in favor of having one `uppy-server.test.js` file for every `uppy-server.js` file in the same directory. That makes it a) easier to spot the test when doing CMD+P in Atom and b) spot which files are missing tests just by looking at the directory. `__mocks__` and `__fixtures__` still get their own subdir as to not overly pollute the directory.

This mostly is a matter of taste and as you see mine changes over time - So if you agree, let's fix it, but if you don't: all good, this is your project now 👌